cypress/integration/createBoard.spec.js

```diff
@@ -0,0 +1,134 @@
+        cy.get(board.nextButton).click();
+        cy.get(board.nextButton).click();
+        cy.get(board.cancelXButton).click();
+    })
+
+    it("createScrumBoardValid", ()=>{
+        cy.get(organization.organizationList).click();
+        cy.get(board.xButton).click();
+        cy.get(board.addBoardField).click();
+        cy.get(board.addBoardTitle).type(name);
+        cy.get(board.nextButton).click();
+        cy.get(board.scrumBoardButton).click();
+        cy.get(board.nextButton).click();
+        cy.get(board.nextButton).click();
+        cy.get(board.nextButton).click();
+        cy.get(board.boardList).click();
+        cy.get(board.boardSettingsButton, {timeout : 3000}).eq(8).click();
+        cy.get(board.deleteBoardButton).scrollIntoView().click();
+        cy.get(board.saveButton).click();
+    })
+
+    it("createKanbanBoardValid", ()=>{
+        cy.get(organization.organizationList).click();
+        cy.get(board.xButton).click();
+        cy.get(board.addBoardField).click();
+        cy.get(board.addBoardTitle).type(name);
+        cy.get(board.nextButton).click();
+        cy.get(board.kanbanBoardButton).eq(1).click();
+        cy.get(board.nextButton).click();
+        cy.get(board.nextButton).click();
+        cy.get(board.nextButton).click({force:true});
+        cy.get(board.boardList).click();
+        cy.get(board.boardSettingsButton, {timeout : 3000}).eq(8).click();
+        cy.get(board.deleteBoardButton).scrollIntoView().click();
+        cy.get(board.saveButton).click();
+    })
+
+    it("changeBoardType", ()=>{
+        cy.get(organization.organizationList).click();
+        cy.get(board.xButton).click();
+        cy.get(board.addBoardButton).click();
+        cy.get(board.addBoardTitle).type(name);
+        cy.get(board.nextButton).click();
+        cy.get(board.scrumBoardButton).click();
+        cy.get(board.nextButton).click();
+        cy.get(board.nextButton).click();
+        cy.get(board.nextButton).click({force:true});
+        cy.get(board.boardList).click();
+        cy.get(board.boardSettingsButton, {timeout : 3000}).eq(8).click();
+        cy.get(board.boardTypeButton).scrollIntoView().click();
+        cy.get(board.boardTypeChecbox).eq(1).click();
+        cy.get(board.saveButton).click();
+        cy.get(board.deleteBoardButton, {timeout: 3000}).scrollIntoView().click();
+        cy.get(board.saveButton).click();
+    })
+
+    it("updateBoardTitle", ()=>{
+        cy.get(organization.organizationList).click();
+        cy.get(board.xButton).click();
+        cy.get(board.addBoardButton).click();
+        cy.get(board.addBoardTitle).type(name);
+        cy.get(board.nextButton).click();
+        cy.get(board.scrumBoardButton).click();
+        cy.get(board.nextButton).click();
+        cy.get(board.nextButton).click();
+        cy.get(board.nextButton).click({force:true});
+        cy.get(board.boardList).click();
+        cy.get(board.boardSettingsButton, {timeout : 3000}).eq(8).click();
```

Comment:
Deleted timeouts. It was left over from some previous code adjustments. Eq(8) is because it there is no better option for getting element, and it is last element in a array.